docs/content/how-to/creating-catalogs.md

```diff
@@ -127,6 +132,11 @@ After `spark-sql` is started, you can switch to the `default` database of the `p
 USE paimon.default;
 ```
 
+If you are using a object storage , and you don't want that the location of paimon table/database is accessed by the filesystem of hive,
```

Comment:
Maybe we can just copy these sentence to after `tab`? So both Flink and Spark can contain it.